tests/Integration/Command/DependencyResolverCommandTest.php

```diff
@@ -82,13 +82,31 @@ public function testMissingArgument()
     /**
      * @dataProvider workingCasesToTest
      */
-    public function testWorkingCases($options, $expected)
+    public function testWorkingCases($options, $expectedRequire, $expectedRepositories)
     {
         $this->configureClient();
         $output = $this->commandTester->execute($options);
         $this->assertEquals(0, $output);
+
+        // Build composer.json
+        $compose = [];
+        if (!empty($expectedRepositories)) {
+            $composeRepositories = [];
+            foreach ($expectedRepositories as $repoFullName => $private) {
+                $composeRepositories[] = [
+                    'type' => 'vcs',
+                    'url' => "https://github.com/${repoFullName}",
+                    'no-api' => !$private
```

Comment:
`no-api` is constant `true` in the implementation. This may get fail depending on your test data.